BlackPearl.Controls.Library/Control/MultiSelectCombobox.xaml.cs

```diff
@@ -289,9 +299,28 @@ private void MultiChoiceControl_LostFocus(object sender, RoutedEventArgs e)
         {
             //Remove all invalid texts from
             RemoveInvalidTexts();
+        }
 
-            //Hide drop-down
```

Comment:
Instead of removing this, hide it conditionally.
Like 
```
if (!popup.IsKeyboardFocusWithin)
            {
                HideSuggestionDropDown();
            }
```